sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -143,6 +149,117 @@ internal class ViewAnnotationManagerImpl(
     return renderThread.viewAnnotationMode
   }
 
+  override fun showAnnotations(
+    annotations: List<View>,
+    edgeInsets: EdgeInsets?,
+    bearing: Double?,
+    pitch: Double?,
+    animationDurationMs: Long
+  ) {
+    if (annotations.isNotEmpty()) {
+      val viewAnnotationOptions = annotations.mapNotNull {
```

Comment:
should it just `map`? as I can see `annotations: List<View>` should contain only not null views.

Reply:
`mapNotNull` works on transformed result and if its not null, then adds to the list. i.e. if `getViewAnnotationOptionsByView ` produces null result, then it will not be added to resulting list.